app/assets/stylesheets/admin.scss

```diff
@@ -62,10 +62,6 @@ $sidebar-active: #f4fcd0;
     color: #000;
     height: auto;
 
-    @include breakpoint(medium) {
-      box-shadow: 0 2px 2px #eee;
-    }
-
```

Comment:
I'm fine with this change :ok_hand:. Would it be possible to know the reason (it would be awesome if it were explained in the commit message!)? I'm asking because I wouldn't know what to say if somebody else opens a pull request and adds these lines again :roll_eyes:.

Reply:
I don't see the need, but I've added it to the commit description. 😅 The reason is that it is only decorative and has been deleted because it was also deleted in the front views.